server/client_manager.py

```diff
@@ -758,6 +759,9 @@ def change_area(self, area, password=""):
                 raise ClientError(
                     f"Failed to enter [{area.id}] {area.name}: User already in specified area."
                 )
+            # If they're forced to follow, no escape.
+            if self.forced_to_follow and self.following.area != area:
+                raise ClientError("You can't escape when you've been forced to follow someone!")
```

Comment:
Make sure self.following exists before checking their area, otherwise there will be an error if the following client DC's